lxd/instance/drivers/driver_qemu_templates.go

```diff
@@ -357,6 +358,28 @@ multifunction = "on"
 {{- end }}
 `))
 
+var qemuVirtioFSDriveConfig = template.Must(template.New("qemuVirtioFSDriveConfig").Parse(`
```

Comment:
Kinda confused by this one, why isn't it modeled after qemuDriveDir?

Reply:
What do you mean? Is the order wrong?

Reply:
No but I wasn't expecting us to get any new template as a result of this.

Everywhere we do 9p now, we need to do 9p and virtiofs.
So I was instead expecting the two existing templates:
 - qemuDriveConfig
 - qemuDriveDir

To be altered to do both of the protocols.